docs/core/testing/selective-unit-tests.md

```diff
@@ -87,12 +88,54 @@ In the code example, the defined traits with keys `Category` and `Priority` can
 | Expression | Result |
 | ---------- | ------ |
 | `dotnet test --filter XUnit` | Runs tests whose `FullyQualifiedName` contains `XUnit`.  Available in `vstest 15.1+`. |
-| `dotnet test --filter Category=bvt` | Runs tests which have `[Trait("Category", "bvt")]`. |
+| `dotnet test --filter Category=CategoryA` | Runs tests which have `[Trait("Category", "CategoryA")]`. |
+
+**Using conditional operators | and &amp;**
+
+| Expression | Result |
+| ---------- | ------ |
+| <code>dotnet test --filter "FullyQualifiedName~TestClass1&#124;Category=CategoryA"</code> | Runs tests which has `TestClass1` in `FullyQualifiedName` **or** `Category` is `CategoryA`. |
+| `dotnet test --filter "FullyQualifiedName~TestClass1&Category=CategoryA"` | Runs tests which has `TestClass1` in `FullyQualifiedName` **and** `Category` is `CategoryA`. |
+| <code>dotnet test --filter "(FullyQualifiedName~TestClass1&Category=CategoryA)&#124;Priority=1"</code> | Runs tests which have either `FullyQualifiedName` containing `TestClass1` **and** `Category` is `CategoryA` **or** `Priority` is 1. |
+
+## NUnit
+
+```csharp
+using NUnit.Framework;
+
+namespace NUnitNamespace
+{
+    public class UnitTest1
+    {
+      [Category("CategoryA")]
+      [Property("Priority", 1)]
+      [Test]
+      public void TestMethod1()
+      {
+      }
+
+      [Property("Priority", 2)]
+      [Test]
+      public void TestMethod2()
+      {
+      }
+    }
+}
+```
```

Comment:
I think this code should consistently use 4 spaces for indentation, instead of using 4 spaces for the first level and 2 spaces for the second level, as it is now.